scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/scheme/TypedParquetTupleScheme.scala

```diff
@@ -102,7 +125,7 @@ trait ParquetWriteSupport[T] extends WriteSupport[T] {
  * @param fp filter predicate
  * @tparam T tuple value type
  */
-class TypedParquetTupleScheme[T](val readSupport: Class[_], val writeSupport: Class[_],
+class TypedParquetTupleScheme[T](val readSupport: ParquetReadSupport[T], val writeSupport: ParquetWriteSupport[T],
```

Comment:
Not sure I understand this well. TypedParquetTupleScheme extends Scheme which implements java serializable and both ParquetReadSupport and WriteSupport have implemented serializable interface.